### DISCUSSION TITLE
[推荐]总时数11451.4小时

### DISCUSSION THREAD

**Discussion by muxi0330** [share121/inter-knot#811] [其他]
![096b4f2](https://github.com/user-attachments/assets/9ccd62e3-10de-4d79-b2b3-46aeb7939e37)
喜欢黄油的可以看看我下面的强烈推荐O(∩_∩)O哈哈   ~~~        

绝区零
ZZZ
Zenless Zone Zero

不定时更新黄油资讯
请奖励这条评测，我要点数去买各种各样的骚逼头像和背景，让我一冲到底，奥利给

**Comment by Fzy85** [comment]
哞？真的吗

**Comment by aaa00073** [comment]
这个游戏怎么下载啊，手机可以玩吗，为什么我在手机商店搜不到，为什么解压不了，为什么解压完了文件打不开了？

**Comment by match-fire** [comment]
是你吗牛至精灵(

**Comment by f4reto** [comment]
炎枪冲锋！！

**Comment by capacitor1** [comment]
> Reply::这个游戏怎么下载啊，手机可以玩吗，为什么我在手机商店...

doge
诶呀，我下载了这个东西，怎么这么大啊
这里面这一堆blk文件是啥？？打不开啊？这usm后缀的什么玩意啊？？占这么大体积我就删了啊！！
，
，
我删完了啊，，，怎么运行不来了啊？
，，刚刚被网友劝导，有问题要自己百度，，我就去百度了，pck文件怎么打开，怎么怎么复杂啊？？
玩个游戏怎么怎么难啊楼主是不是在故意骗人，，
dogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedogedoge

**Reply by match-fire** [reply]
6

**Reply by 0d00210** [reply]
血压上来了

**Comment by cyduan** [comment]
这个委托我接了
